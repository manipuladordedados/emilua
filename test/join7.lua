-- This test ensures join() detects EDEADLK

fib = spawn(function()
    fib:join()
end)
