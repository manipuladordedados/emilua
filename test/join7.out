Fiber 0x1 panicked: 'Resource deadlock avoided'
stack traceback:
	[string "?"]: in function 'join'
	input.lua:4: in function <input.lua:3>
